src/index.html

```diff
@@ -11,7 +11,128 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form>
+      <h1> FORM </h1>
+      <p> Будь ласка заповність форму</p>
+      <script type="text/javascript" src="./main.js"></script>
```

Comment:
I would recommend placing the script file at the end of body section (just before closing tag `</body>`)